community/security/src/main/resources/META-INF/services/org.neo4j.kernel.api.security.AuthManager$Factory

```diff
@@ -0,0 +1 @@
+org.neo4j.server.security.auth.BasicAuthManagerFactory
```

Comment:
It specifies a class that provides the service org.neo4j.kernel.api.security.AuthManager.Factory